example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -345,15 +364,28 @@ describe('The S3 Ingest Granules workflow', () => {
       await accessTokensModel.delete({ accessToken });
     });
 
-    it('has expected payload', () => {
-      expect(granule.published).toBe(true);
+    xit('has expected payload', () => {
```

Comment:
I don't think you want this disabled

Reply:
Fixed. I swear, you catch either an `.only` or `xit` in every PR I issue.